aptos-move/framework/aptos-experimental/sources/trading/market/market.move

```diff
@@ -98,7 +107,11 @@ module aptos_experimental::market {
             // Incremental fill id for matched orders
             next_fill_id: u64,
             config: MarketConfig,
-            order_book: OrderBook<M>
+            order_book: OrderBook<M>,
+            /// Pre cancellation tracker for the market, it is wrapped inside a table
+            /// as otherwise any insertion/deletion from the tracker would cause conflict
+            /// with the order book.
```

Comment:
Can we move the tracker to global storage instead to avoid this table?

Reply:
This is a struct, has no signer or objects, so this is only way to use global storage?

Reply:
Ah yes... It's almost like we need an equivalent of a Box<T> in rust...